cilium/l7policy.h

```diff
@@ -18,8 +18,8 @@ namespace Cilium {
  * All Cilium L7 filter stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_CILIUM_STATS(COUNTER)                                                                  \
-  COUNTER(access_denied)                                                                                \
+#define ALL_CILIUM_STATS(COUNTER)	\
```

Comment:
there's still extra whitespace? is this necessary/formatted?

Reply:
It is customary for the backslash within a macro definition to be some whitespace away, IMO.

Formatting if off for the macros (`// clang-format off` right above).